Automattic/newspack-theme
# Add styles for single article author bio.

### All Submissions:


### Changes proposed in this Pull Request:

This PR roughs in the styles for the author bios at the bottom of each story.

Still outstanding:

* Additional fields -- right now, the job title ("Staff Writer") is hard coded. We need to finalize what fields we might need (title, additional social media like Twitter), and add dynamic fields. I'm also wondering if displaying email should be optional (since it's a required field in WordPress).
* Should showing the bio at all be an option (say, for shorter stories). 
* Co-Authors Plus -- assuming we go with this plugin, this code will need to be updated to include its custom functions, with the standard ones as a fallback.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build` (edited to add the build).
2. View a story where the assigned author has a description (right now, the theme copies Twenty Nineteen and requires the author to have a description before the profile is displayed at the bottom). 
3. Confirm that the styles match those mocked up in 'Style A': https://cloudup.com/ilw9quyPhje

### Other information:
